src/pal/src/init/pal.cpp

```diff
@@ -436,7 +436,7 @@ Initialize(
         if (NO_ERROR != palError)
         {
             ERROR("Unable to create initial process and thread objects\n");
-            goto CLEANUP4;
+            goto CLEANUP2;
```

Comment:
I agree that the code is ugly and I dislike use of gotos myself. A cleanup of this would be great, but it would require a nontrivial refactoring of the initialization code.